css/global.css

```diff
@@ -1,3 +1,4 @@
-div:has(div.update-components-celebration) {
+.update-components-celebration:has(div.feed-shared-celebration-image), .update-components-mini-update-v2:has(div.feed-shared-celebration-image) {
```

Comment:
The second half of this (starting at `.update-components-mini-update-v2`) is needed to hide reshares of celebration posts.

Reply:
This block of code looks for a parent element with a specified child element and removes it from the DOM.